client/components/site-selector-modal/index.jsx

```diff
@@ -58,9 +66,9 @@ var SiteSelectorModal = React.createClass( {
 	getMainLink: function() {
 		var url = this.props.getMainUrl && this.props.getMainUrl( this.state.site );
 
-		return url ?
-			<a href={ url } className="button is-primary">{ this.props.mainActionLabel }</a> :
-			{ action: 'mainAction', label: this.props.mainActionLabel, isPrimary: true };
+		return url
+			? <a href={ url } className="button is-primary">{ this.props.mainActionLabel }</a>
```

Comment:
Since you are here, you may want to swap this for a real `<Button primary>` component.